setup.py

```diff
@@ -3,8 +3,15 @@
 """
 import os
 import re
-from distutils.core import setup
 
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+
+with open("README.rst", "r") as fh:
```

Comment:
Most of the file uses single quotes, so please change this to single quotes. @morenol